drivers/gpio/Kconfig.rt1718s

```diff
@@ -0,0 +1,35 @@
+# RT1718S driver configuration options
+
+# Copyright 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig GPIO_RT1718S
+	bool "RT1718S I2C-based TCPC chip with GPIOs"
+	default y
+	depends on DT_HAS_RICHTEK_RT1718S_GPIO_PORT_ENABLED
+	help
+	  Enable driver GPIO for RT1718S I2C-based TCPC chip.
+
+if GPIO_RT1718S
+
+config RT1718S_INIT_PRIORITY
+	int "RT1718S GPIO init priority"
+	default 50
```

Comment:
I2C defaults to 50, should this default to something higher?

Reply:
Good question. I've considered this. As you pointed config `I2C_INIT_PRIORITY` and  `KERNEL_INIT_PRIORITY_DEVICE` equals 50 by default, but it is a different level of initialization. However, the I2C controller drivers usually use a different level `PRE_KERNEL`, but now I can see that is not a rule.